lib/apivore/swagger_checker.rb

```diff
@@ -42,10 +44,12 @@ def base_path
       @swagger.base_path
     end
 
-    attr_reader :swagger_path, :mappings, :swagger
+    attr_reader :swagger_path, :untested_mappings, :swagger
 
     private
 
+    attr_reader :mappings
```

Comment:
Is there a particular reason why this is now private? I can't see an issue with it, just curious.

LGTM otherwise